### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by BauRo**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/BauRo/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
```
Comment (BauRo) [inline ReadMe.md]: testing comment !resolve

Comment (BauRo) [inline ReadMe.md]: !fixed

Comment (BauRo) [inline ReadMe.md]: !fix

Comment (BauRo) [inline ReadMe.md]: !fixed

**Pull request merged**